proto/cosmos/gov/v1beta2/genesis.proto

```diff
@@ -5,18 +5,18 @@ package cosmos.gov.v1beta2;
 import "gogoproto/gogo.proto";
 import "cosmos/gov/v1beta2/gov.proto";
 
-option go_package = "github.com/cosmos/cosmos-sdk/x/gov/types";
+option go_package = "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta2";
 
 // GenesisState defines the gov module's genesis state.
 message GenesisState {
   // starting_proposal_id is the ID of the starting proposal.
   uint64 starting_proposal_id = 1 [(gogoproto.moretags) = "yaml:\"starting_proposal_id\""];
   // deposits defines all the deposits present at genesis.
-  repeated Deposit deposits = 2 [(gogoproto.castrepeated) = "Deposits", (gogoproto.nullable) = false];
+  repeated Deposit deposits = 2 [(gogoproto.nullable) = false];
```

Comment:
I think we're moving away from castrepeated (to double-check with the pulsar WG), so this LGTM.

But just in case we're not, i believe this works too:

```proto
repeated Deposit deposits = 2 [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/x/gov/types.Deposits", (gogoproto.nullable) = false];
```

Reply:
removed it